src/h5web/metadata-viewer/AttributesTable.tsx

```diff
@@ -17,14 +17,16 @@ function AttributesTable(props: Props): ReactElement {
     <table className={styles.table}>
       <thead>
         <tr>
-          <th colSpan={2}>Attributes</th>
+          <th scope="col" colSpan={2}>
+            Attributes
+          </th>
         </tr>
       </thead>
       <tbody>
         {attributes.map(
           ({ name, value }: HDF5Attribute): ReactElement => (
             <tr key={name}>
-              <th>{name}</th>
+              <th scope="row">{name}</th>
```

Comment:
I had async troubles in the tests. This fix was not the issue after all, but since it doesn't hurt, I left it... :D